core/ui/EditorToolbar/picture-dropdown.tid

```diff
@@ -4,8 +4,44 @@ title: $:/core/ui/EditorToolbar/picture-dropdown
 [img[$(imageTitle)$]]
 \end
 
+\define insert-external-image()
```

Comment:
OK, fair enough, just though it might be useful for someone else since it fits my use case though it certainly could be improved as you suggested by displaying the image preview.

Feel free to close this anytime, and thanks for the feedback